bin/move-to-go

```diff
@@ -38,7 +40,15 @@ class MoveToGoCommandLine < Thor
     end
 
     desc "new <PROJECT> <SOURCE>", "Creates a new migration project with a specifed name and source"
+    option(:ignore_version,
+           :desc => "Ignore version check",
+           :type => :boolean,
+           :required => false)
     def new(project, source)
+        if options.ignore_version.nil?
+            verify_version()
+        end
```

Comment:
Blir inte detta tvärtom? Som default vill vi att move-to-go inte ska köra med gamla versioner. Sen är jag inte helt nöjd med namnet, vad tror du om:

`move-to-go run --force-old-version`

eller 

`move-to-go run --force-current-version`

Sen är det bra om namnen på argumentet (--ignore-version eller -force-current-version, etc) och den interna funktionen (verify_version) har med varandra att göra.